bigbluebutton-client/src/org/bigbluebutton/modules/whiteboard/views/WhiteboardCanvas.as

```diff
@@ -297,15 +311,11 @@ package org.bigbluebutton.modules.whiteboard.views {
 			if (doesContainCursor(cursor)) this.cursorObjectHolder.rawChildren.removeChild(cursor);
 		}
 		
-		public function removeAllCursors():void {
-			var newCursorHolder:Canvas = new Canvas;
-			newCursorHolder.height = cursorObjectHolder.height;
-			newCursorHolder.width = cursorObjectHolder.width;
-			newCursorHolder.tabFocusEnabled = false;
-			
-			addChildAt(newCursorHolder, getChildIndex(cursorObjectHolder));
-			removeChild(cursorObjectHolder);
-			cursorObjectHolder = newCursorHolder;
+		public function removeAllCursorChildren():void {

```

Comment:
The cursor clearing approach should match whatever is selected for the annotation clearing. I'm not 100% sure which approach is quicker, but my gut says that creating a new one might be quicker.

Reply:
I kept `removeAllCursorChildren`. All other merge issues have been resolved.